entrypoint.sh

```diff
@@ -2,8 +2,13 @@
 set -e
 set -o pipefail
 
+if [[ $GITLAB_REGISTRY_GENERATE_INTERNAL_CERTIFICATES == 'true' ]]; then
```

Comment:
It would be more accurate to move this under `assets/runtime/function` and add the env to `
/assets/runtime/env-defaults`
1. Create an own function for your certificate generation 
2. Provide the env in in env-defaults

Also to be backwards compatible it should set to `false` by default.

Reply:
Perfect. Done. :heavy_check_mark: